src/app/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -32,12 +32,30 @@ import type {
   PDFViewerOptions,
   ZoomScale
 } from './typings';
-import { PDFSinglePageViewer } from 'pdfjs-dist/web/pdf_viewer';
+import { GlobalWorkerOptions, VerbosityLevel, getDocument } from 'pdfjs-dist';
+
 
 if (!isSSR()) {
-  assign(PDFJS, 'verbosity', PDFJS.VerbosityLevel.INFOS);
+  assign(PDFJS, 'verbosity', VerbosityLevel.INFOS);
 }
 
+// @ts-expect-error This does not exist outside of polyfill which this is doing
+if (typeof Promise.withResolvers === 'undefined') {
+  if (window) {
+    // @ts-expect-error This does not exist outside of polyfill which this is doing
```

Comment:
Please combine the ifs and keep only one comment

Reply:
@VadimDez I consolidated the ifs statements. The comments on the other hand are a different story... because we're doing an assignment in one line and a type check in another line, Typescript will flag either reference to `Promise.withResolvers` as being not defined in the `PromiseConstructor` (because this is a pollyfill and we need to suppress both messages one way or another). So if you don't want there to be two `@ts-expect-error` statements, I could replace one of them with a `@ts-ignore`, which will still be self-documenting, but less likely to be confused as a random duplicate comment.